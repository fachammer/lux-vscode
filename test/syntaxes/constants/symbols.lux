a
symbol
a-1
a?
a!
<symbol>