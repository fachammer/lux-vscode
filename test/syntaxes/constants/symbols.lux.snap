>a
#^ source.lux meta.symbol.clojure
>symbol
#^^^^^^ source.lux meta.symbol.clojure
>a-1
#^^^ source.lux meta.symbol.clojure
>a?
#^^ source.lux meta.symbol.clojure
>a!
#^^ source.lux meta.symbol.clojure
><symbol>
#^^^^^^^^ source.lux meta.symbol.clojure